asf_heat_pump_suitability/pipeline/prepare_features/lat_lon.py

```diff
@@ -39,7 +39,7 @@ def generate_gdf_uprn_coords(
     if not usecols:
         usecols = ["*"]
     else:
-        for col in ["X_COORDINATE", "Y_COORDINATE"]:
+        for col in [x_col, y_col]:
```

Comment:
This line still worked before, but this is more correct since we take these columns as inputs to the function.